source/guides/core-concepts/test-runner.md

```diff
@@ -91,6 +91,8 @@ The image below shows that our application is displaying at `1000px` width, `660
 
 {% imgTag /img/guides/errors.png "Errors" %}
 
+*Note: Internally, AUT uses iframe. It can sometimes cause unexpected {% url "behaviors like this." /api/commands/window.html#Cypress-uses-2-different-windows %}*
```

Comment:
```suggestion
*Note: Internally, the AUT renders within an iframe. This can sometimes cause unexpected behaviors {% url "explained here." /api/commands/window.html#Cypress-uses-2-different-windows %}*
```